lib/aggregators/top-balances.ts

```diff
@@ -0,0 +1,31 @@
+import Aggregator from './aggregator';
+import { getUnlockedSupply, getTopBalances } from '../core-db-pg/queries';
+import { microStacksToStacks } from '../utils';
+
+export interface TopBalanceAccount {
+  address: string;
+  balance: string;
+  distribution: number;
+}
+
+class TopBalancesAggregator extends Aggregator {
+  static key(count: number) {
+    return `${this.name}:${count}`;
+  }
+
+  static expiry() {
+    return 10 * 60; // 10 minutes
+  }
+
+  static async setter(count: number): Promise<TopBalanceAccount[]> {
+    const { unlockedSupply } = await getUnlockedSupply();
+    const topBalances = await getTopBalances(count);
+    return topBalances.map(account => ({
```

Comment:
Can we also get a `stacksAddress` attribute in here? `c32check.b58ToC32(account.address)`

Reply:
Done